### PR TITLE
Add scripts to setup and update local feeds

### PR DESCRIPTION
**Pull request by JunTaoLuo**
Addressing https://github.com/grpc/grpc-dotnet/issues/45. This is built on top of the current PR to migrate code to grpc/grpc-dotnet since it relies on the dependencies.props mechanism. I'll rebase these changes once that PR goes through.

### CODE REVIEW

File: build/upgrade-deps.ps1 [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+function Ensure-Dir([string]$path) {
+    if (!(Test-Path $path -PathType Container)) {
+        New-Item -ItemType Directory -Force -Path $path | Out-Null
+    }
+}
+
+# Main
+
+# Download manifests
+Ensure-Dir $TempDir
```
Comment (JunTaoLuo) [inline build/upgrade-deps.ps1]: Hmm I should add a bit of logging so it doesn't look like the scripts hung.

**Comment by JamesNK** [comment]
The issue I have with committing large files into the repository is they are part of the history forever. If we update Grpc.Core 10 times then the repo is 500 megs in size.

**Comment by JunTaoLuo** [comment]
We could just run the script at the beginning of each build and periodically rerun it locally. However, their package versions do not update between builds so we'll need to clear our caches. I would like to see a build number suffix being added though.

**Comment by JunTaoLuo** [comment]
Closed in favour of https://github.com/grpc/grpc-dotnet/pull/57.

**Pull request closed without merging**